test/resources/disabled-init-test-workspace.yaml

```diff
@@ -0,0 +1,16 @@
+kind: DevWorkspace
+apiVersion: workspace.devfile.io/v1alpha2
+metadata:
+  name: disabled-init-test
```

Comment:
This file and `custom-init-test-workspace.yaml` only differ by name. Is it possible to change the name inside test code to avoid duplication?